code/reports/SubsiteReportWrapper.php

```diff
@@ -35,35 +45,44 @@ public function parameterFields()
         return $fields;
     }
 
-    ///////////////////////////////////////////////////////////////////////////////////////////
-    // Columns
-
+    /**
+     * @return array
+     */
     public function columns()
     {
         $columns = parent::columns();
-        $columns['Subsite.Title'] = "Subsite";
+        $columns['Subsite.Title'] = Subsite::class;
```

Comment:
I assume this was the upgrader, but it should remain as it was since it's a report column

Reply:
Probably need a good `Subsite::class` text review to make sure all of these get caught.